src/components/SQForm/SQForm.tsx

```diff
@@ -24,12 +50,15 @@ function SQForm({
 
   // HACK: This is a workaround for: https://github.com/mui-org/material-ui-pickers/issues/2112
   // Remove this reset handler when the issue is fixed.
-  const handleReset = () => {
-    document && document.activeElement && document.activeElement.blur();
+  const handleReset = (): void => {
+    document &&
+      document.activeElement &&
+      (document.activeElement as HTMLElement).blur();
   };
 
   const handleSubmit = useDebouncedCallback(
-    (...args) => onSubmit(...args),
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (...args: [any]) => onSubmit(...args),
```

Comment:
I think @jamesgriffith9519 ran into something similar to this the other day and found an alternative to `...args`

Reply:
> I think @jamesgriffith9519 ran into something similar to this the other day and found an alternative to `...args`

I would love to see that, I was stuck with this for a bit, and was nothing seemed to work.

Reply:
> > I think @jamesgriffith9519 ran into something similar to this the other day and found an alternative to `...args`
> 
> I would love to see that, I was stuck with this, and nothing seemed to work.

Sorry for the delay! I didin't notice this until i checked my email and found it, here's the file i was working in and I think it will help with this issue that you are having! https://github.com/SelectQuoteLabs/SQForm/pull/350/files#diff-b2fd4208eb74b575ac362adcbdd31266b1ab12b56a0d0b04bd0bb7d61696d7c4